pkg/portal/cluster/fetcher.go

```diff
@@ -12,14 +12,17 @@ import (
 	"k8s.io/client-go/kubernetes"
 
 	"github.com/Azure/ARO-RP/pkg/api"
+	"github.com/Azure/ARO-RP/pkg/env"
 	"github.com/Azure/ARO-RP/pkg/proxy"
 	"github.com/Azure/ARO-RP/pkg/util/restconfig"
+	"github.com/Azure/ARO-RP/pkg/util/stringutils"
 )
 
 // FetchClient is the interface that the Admin Portal Frontend uses to gather
 // information about clusters. It returns frontend-suitable data structures.
 type FetchClient interface {
 	Nodes(context.Context) (*NodeListInformation, error)
+	VMAllocationStatus(context.Context) (VMAllocationStatus, error)
```

Comment:
Since we have made changes to the interface, can we check to see if it requires generating and committing new  mocks for the interface?

Reply:
We do not have mocks for fetcher right now. Maybe we can generate them in a separate work item.